lib/plain/entities/workflow.ts

```diff
@@ -0,0 +1,128 @@
+import { RawAxiosRequestHeaders } from 'axios'
+import { GetSpaceEnvironmentParams, CollectionProp } from '../../common-types'
+import {
+  CreateWorkflowParams,
+  UpdateWorkflowParams,
+  CompleteWorkflowParams,
+} from '../../entities/workflow'
+import {
+  WorkflowQueryOptions,
+  WorkflowProps,
+  CreateWorkflowProps,
+  UpdateWorkflowProps,
+  DeleteWorkflowParams,
+} from '../../export-types'
+import { OptionalDefaults } from '../wrappers/wrap'
+
+export type WorkflowPlainClientAPI = {
+  /**
+   * Query Workflows with certain filters
+   * @param params entity IDs to identify the Space/Environment, optional query parameters to filter returned Workflows
+   * @returns an object containing the list of Workflows
+   * @throws if the request fails, or the Space/Environment is not found
+   * @example
+   * ```javascript
+   * const results = await client.workflow.getMany({
```

Comment:
I started calling this getMany result values the plural of whatever it is i.e. `workflows`, even tho the return the list of workflow are in `workflows.items`